### DISCUSSION TITLE
Cefsharp.WinForms resize bug

### DISCUSSION THREAD

**Discussion by raphm4044** [cefsharp/CefSharp#4005] [General]
Hello , i have a bug with CefSharp : the ChromiumWebBrowser don't resize correctly

Here is a screen capture :
![image](https://user-images.githubusercontent.com/81994075/153873496-0fc6e1ff-ed4a-4105-b297-b5a4b7753cf8.png)

PS : I'm using CefSharp 97.1.61 with Windows 11 10.0.22000.493

Thanks 

-RaphMar2021

**Comment by raphm4044** [comment]
* : Sorry for the French Windows

**Reply by amaitland** [reply]
See https://github.com/cefsharp/CefSharp/wiki/General-Usage#high-dpi-displayssupport
